calibrators/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/camera_calibrator.py

```diff
@@ -850,7 +964,9 @@ def process_detection_results(self, img: np.array, detection: BoardDetection, im
             self.image_view.set_detection_ordered_points(None)
 
             self.raw_detection_label.setText("Detected: False")
-            self.raw_linear_error_rms_label.setText("Linear error rms:")
+            self.raw_linear_error_rows_rms_label.setText("Linear error rows rms:")
+            self.raw_linear_error_cols_rms_label.setText("Linear error cols rms:")
+            self.aspect_ratio_label.setText("Aspect Ratio:")
```

Comment:
let's avoid mixing cases

Reply:
im not sure what to do here, sorry.

Reply:
Seems I made a mistake when selecting the lines, but I meant cases like: `Aspect Ratio:` to `Aspect ratio:`

Reply:
f2ad5cce4cffc72284d0b575020c40851475872e